extras/big-sur/README.md

```diff
@@ -62,10 +62,11 @@ You will also need to ensure you have a few NVRAM variables set:
 
 * **`NVRAM` -> `Add` -> `7C436110-AB2A-4BBB-A880-FE41995C9F82`**:
   * `boot-args`:
-    * `-lilubetaall` (Enables Lilu and plugins on beta macOS versions)
+    * ~~`-lilubetaall`~~
       * Newest builds of Lilu and plugins do not require this boot-arg
```

Comment:
```suggestion
      * Newest builds of Lilu and most plugins do not require this boot-arg
```
Stuff like SystemProfilerMemoryFixup aren't updated yet but it doesn't work on BS yet anyways